src/mlpack/tests/gmm_test.cpp

```diff
@@ -352,48 +353,62 @@ BOOST_AUTO_TEST_CASE(GMMTrainEMMultipleGaussiansWithProbability)
   // be a part of this mixture.
 
   // First Gaussian (d1).
-  BOOST_REQUIRE_SMALL(g.Weights()[sortedIndices[0]] - 0.2, 0.1);
+  REQUIRE(g.Weights()[sortedIndices[0]] - 0.2 == Approx(0.0).margin(0.1));
 
   for (size_t i = 0; i < 3; ++i)
-    BOOST_REQUIRE_SMALL((g.Component(sortedIndices[0]).Mean()[i]
-        - d1.Mean()[i]), 0.4);
+  REQUIRE((g.Component(sortedIndices[0]).Mean()[i]
+      - d1.Mean()[i]) == Approx(0.0).margin(0.4));
 
   for (size_t row = 0; row < 3; row++)
+  {
     for (size_t col = 0; col < 3; col++)
-      BOOST_REQUIRE_SMALL((g.Component(sortedIndices[0]).Covariance()(row, col)
-          - d1.Covariance()(row, col)), 0.7); // Big tolerance!  Lots of noise.
+    {
+       // Big tolerance!  Lots of noise.
+      REQUIRE((g.Component(sortedIndices[0]).Covariance()(row, col)
+          - d1.Covariance()(row, col)) == Approx(0.0).margin(0.7));
+    }    
```

Comment:
```suggestion
    }
```

This should fix the style issue.